map-storage/src/index.ts

```diff
@@ -100,7 +101,7 @@ app.get("*.wam", (req, res, next) => {
     // Maybe the map is already in memory (in case this map is edited by the current map storage)
     const gameMap = mapsManager.getGameMap(key);
     if (gameMap) {
-        res.send(gameMap.getWam());
+        res.send(wamFileMigration.migrate(gameMap.getWam()));
```

Comment:
If the map is already in memory, it is already migrated. No need to migrate here.